### ISSUE TITLE
Incompatible types in assignment

### ISSUE THREAD

**Issue by parampatidar** [Instagram/MonkeyType#220]
To setting up l[ocal dev environment](https://github.com/Instagram/MonkeyType/blob/master/CONTRIBUTING.rst#local-dev-environment) when I run `pipenv run mypy monkeytype` to typecheck I get following traceback:
```python-trackback
monkeytype/tracing.py:81:31: error: Incompatible types in assignment (expression has type "object", variable has type "Optional[type]")
Found 1 error in 1 file (checked 14 source files)
```

**Comment by carljm** [comment]
Must be a change in a newer mypy version, I'll take a look. Thanks for the report!

**Comment by carljm** [comment]
This should be fixed in f584b595e20d3d459786f1801964641c13458b89
